### PR TITLE
Added settings autosave and removed save button

### PR DESCRIPTION
**Pull request by mikevich**
Simple change.

Changed the event listener from listening for the form to submit, to listening for any change to the form. Also removed the Save button in the HTML.

### CODE REVIEW

File: src/options.js [hunk 1]
```diff
@@ -68,12 +68,16 @@ function saveOptions(event) {
 
   chrome.storage.sync.set(
     options,
-    () => showAlert("success", "Settings saved!", 2000)
+    () => document.getElementById("saveMessage").innerHTML("Saving...")
```
[the remark this answers is not in the document]
Comment (mikevich) [reply src/options.js]: Implemented!

File: src/options.js [hunk 2]
```diff
@@ -66,14 +66,25 @@ function saveOptions(event) {
...
+
+  chrome.storage.sync.set(options).then(success, error)
+}
+
+function success() {
+  document.getElementById("saveMessage").innerHTML("Saved.")
+}
+
+function error() {
+  showAlert("danger", "Failed to save settings!", 2000)
```
[the remark this answers is not in the document]
Comment (mikevich) [reply src/options.js]: Done

File: src/options.html [hunk 3]
```diff
@@ -23,6 +23,11 @@
       form > *:not(:first-child) {
         margin-top: 2rem;
       }
+
+      .save_alert {
```
Comment (hmitgang) [inline src/options.html:27]:
```suggestion
      .save-alert {
```

**Review by hmitgang — APPROVED** [review]
LGTM!

**Pull request merged**